pkg/tnf/handlers/bootconfig/bootconfig.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2021 Red Hat, Inc.
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package bootconfig
+
+import (
+	"time"
+
+	"github.com/test-network-function/test-network-function/pkg/tnf"
+	"github.com/test-network-function/test-network-function/pkg/tnf/identifier"
+	"github.com/test-network-function/test-network-function/pkg/tnf/reel"
+)
+
+const (
+	successfulOutputRegex = `(?s).+`
+)
+
+// BootConfig holds information regarding boot config in /boot directory.
+type BootConfig struct {
+	bootConfig string // Output variable that stores the boot config as specified in the /boot directory
+	result     int
+	timeout    time.Duration
+	args       []string
+}
+
+// NewBootConfig creates a BootConfig tnf.Test.
+func NewBootConfig(timeout time.Duration, mcName string) *BootConfig {
+	return &BootConfig{
+		timeout: timeout,
+		result:  tnf.ERROR,
+		args: []string{
+			"echo \"cat /host/boot/loader/entries/\\`ls -l /host/boot/loader/entries/ | sort -r -k 9 | head -n 1 | cut -d\\\" \\\" -f9 -\\`\" | oc debug -q node/crc-l6qvn-master-0",
```

Comment:
I am a bit confused by what this actually does.  It looks like it is hard-coded for a specific CRC `Node` though, and will not work against a stock cluster.  Not every cluster will have `node/crc-l6qvn-master-0` installed.

Can you provide a way to make the node name configurable?